sky/shell/android/org/domokit/sky/shell/SkyApplication.java

```diff
@@ -74,6 +76,22 @@ public void connectToService(Context context, Core core, MessagePipeHandle pipe)
             }
         });
 
+        registry.register(org.chromium.mojom.updater.UpdateService.MANAGER.getName(),
+                          new ServiceFactory() {
+            @Override
+            public void connectToService(Context context, Core core, MessagePipeHandle pipe) {
+                org.chromium.mojom.updater.UpdateService.MANAGER.bind(
+                    UpdateService.sCurrentUpdateService, pipe);
```

Comment:
Good question. I changed this so that it binds to a new instance of a class that talks to UpdateService.